cli/src/main/kotlin/commands/EvaluatorCommand.kt

```diff
@@ -209,7 +224,15 @@ class EvaluatorCommand : CliktCommand(name = "evaluate", help = "Evaluate rules
             }
         }
 
-        val evaluator = Evaluator(ortResultInput, packageConfigurationProvider, licenseConfiguration)
+        val copyrightGarbage = copyrightGarbageFile.takeIf { it.isFile }?.readValue<CopyrightGarbage>().orEmpty()
+
+        val licenseInfoResolver = LicenseInfoResolver(
+            provider = DefaultLicenseInfoProvider(ortResultInput, packageConfigurationProvider),
+            copyrightGarbage = copyrightGarbage,
+            archiver = globalOptionsForSubcommands.config.scanner?.archive?.createFileArchiver() ?: FileArchiver.DEFAULT
```

Comment:
> Is the file archiver used?

We don't currently do that, but it could be used from the rules. 

> I wonder whether it is ok to consume the `scanner` configuration from the `evaluator`, what do you think?

Yes, because it's the same in the reporter. We could consider moving this part of the config to a generic place, but that's a different topic.